app/src/components/queue.js

```diff
@@ -0,0 +1,60 @@
+const Bull = require('bull');
+const config = require('config');
+const log = require('npmlog');
+const uuidv4 = require('uuid/v4');
+
+const email = require('./email');
```

Comment:
See earlier comment, don't like this circle.

Reply:
Cyclic dependency refactored out.